src/components/ReportActionItem/MoneyRequestView.js

```diff
@@ -208,8 +247,10 @@ function MoneyRequestView({report, parentReport, policyCategories, shouldShowHor
                         titleStyle={styles.flex1}
                         onPress={() => Navigation.navigate(ROUTES.EDIT_REQUEST.getRoute(report.reportID, CONST.EDIT_REQUEST_FIELD.DESCRIPTION))}
                         wrapperStyle={[styles.pv2, styles.taskDescriptionMenuItem]}
+                        brickRoadIndicator={hasViolations('comment') ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : ''}
                         numberOfLinesTitle={0}
                     />
+                    <ViolationMessages field="comment" />
```

Comment:
This is almost certainly out of scope for feedback, but just to note: This is a case where I don't fully understand why `ViolationMessages` is not part of `MenuItemWithTopDescription` (and the receipt component) as a generic error message instead. Then there could be a simple `errorMessage` prop on the component whose presence can be used to trigger rendering the error message and the red dot.

Reply:
Yeah no idea.